lib/jetstream/api/consumer.ex

```diff
@@ -83,6 +307,21 @@ defmodule Jetstream.API.Consumer do
     end
   end
 
+  @doc """
+  Paged list of known consumers, including their current info.
+
+  ## Examples
+
+      iex> list(gnat, "stream")
+      {:ok, %{total: 2, offset: 0, limit: 1024, consumers: ["consumer1, consumer2"]}}
+
+      iex> list(gnat, "stream2", offset: 10)
+      {:ok, %{total: 12, offset: 10, limit: 1024, consumers: ["cosumer11", "consumer12"]}}
+
+      iex> list(gnat, "wrong_stream")
+      {:error, %{"code" => 404, "description" => "stream not found"}}
+
+  """
   @spec list(Gnat.t(), binary(), offset: non_neg_integer()) ::
```

Comment:
```suggestion
  @spec list(Gnat.t(), binary(), offset :: non_neg_integer()) ::
```

Reply:
This is a keyword list with `offset` key, added the variable name here to make it more readable